src/ringct/rctSigs.cpp

```diff
@@ -717,7 +770,10 @@ namespace rct {
         CHECK_AND_ASSERT_MES(rv.type == RCTTypeFull, false, "verRct called on non-full rctSig");
```

Comment:
I believe this should be:

         CHECK_AND_ASSERT_MES(rv.type == RCTTypeFull || rv.type == RCTTypeFullBulletproof, false, "verRct called on non-full rctSig");

assuming that BPs are allowed for the Full RingCT.